### DISCUSSION TITLE
[a11y] Route Announcements

### DISCUSSION THREAD

**Discussion by adamdbradley** [QwikDev/qwik-evolution#280] [Proposals For Qwik]
### Is your feature request related to a problem?

Improves the accessibility of client-side navigation by announcing route changes to screen readers.

### Describe the solution you'd like

Inspired by https://www.gatsbyjs.com/blog/2019-07-11-user-testing-accessible-client-routing/

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
